lib/msf/core/exploit/sqli/mysqli/boolean_based_blind.rb

```diff
@@ -0,0 +1,52 @@
+#
+#   Boolean-Based Blind SQL injection support for MySQL
+#
+class Msf::Exploit::SQLi::MySQLi::BooleanBasedBlind < Msf::Exploit::SQLi::MySQLi::Common
+  #
+  #   Gets the output of the given SQL query, in a boolean-based blind manner.
+  #   The block given to initialize must return true if querying its parameter
+  #   gave a result, false otherwise.
+  #   @param query [String] The SQL query to execute
+  #   @param output_charset [Range] The range of characters to expect in the output, optional
```

Comment:
I believe it was mentioned that can optimize calls and speed up execution.  Elaborating on the value here on when used in a module will help others understand when to take advantage of this.